frontend/src/app/drive/layout.tsx

```diff
@@ -0,0 +1,96 @@
+import { Switch } from "@headlessui/react";
+import {
+  GlobeIcon,
+  HomeIcon,
+  ShareIcon,
+  TrashIcon,
+  UserIcon,
+} from "lucide-react";
+import type { Metadata } from "next";
+import localFont from "next/font/local";
+import { useLocalStorage } from "usehooks-ts";
+import { ScopeSwitch } from "../../components/common/ScopeSwitch";
+import { SearchBar } from "../../components/SearchBar";
+import "../globals.css";
+
+const geistSans = localFont({
+  src: "../fonts/GeistVF.woff",
+  variable: "--font-geist-sans",
+  weight: "100 900",
+});
+const geistMono = localFont({
+  src: "../fonts/GeistMonoVF.woff",
+  variable: "--font-geist-mono",
+  weight: "100 900",
+});
+
+export const metadata: Metadata = {
+  title: "Auto-Drive",
+  description: "Autonomys Drive",
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="en">
+      <body
+        className={`${geistSans.variable} ${geistMono.variable} antialiased`}
+      >
+        <div className="min-h-screen bg-white px-8">
+          <div className="flex h-screen flex-col bg-white rounded-lg p-6">
+            <header className="flex flex-col md:flex-row gap-4 md:gap-0 items-center justify-between mb-8">
+              <div className="flex items-center space-x-2">
+                <img
+                  src="/autonomys.png"
+                  alt="Auto Drive"
+                  className="w-[2rem] h-[2rem] rounded-full"
+                />
+                <span className="text-xl font-semibold">Auto Drive</span>
+              </div>
+              <div className="flex items-center space-x-4">
+                <span className="text-sm font-medium">Your Files</span>
+                <ScopeSwitch />
+                <span className="text-sm font-medium">Global</span>
+                <div className="md:w-80">
+                  <SearchBar />
+                </div>
+              </div>
+            </header>
+            <div className="flex flex-1 overflow-hidden">
+              <aside className="w-16 md:w-64 space-y-4">
+                <a href="/drive" className="contents">
```

Comment:
In next, especially when navigating to internal page, <Link should be use to avoid a full page reload